### DISCUSSION TITLE
protect not working since unifi update

### DISCUSSION THREAD

**Discussion by shocker42** [briis/unifiprotect#177] [Q&A] [answered]
since the last update on my cloud key 2 + the plugin doesnt work because it doesnt need a port now and it is https

**Comment by briis** [comment, marked as answer]
Please see my answer in [this post](https://github.com/briis/unifiprotect/issues/168)

**Reply by shocker42** [reply]
thanks it is working now

**Comment by xPhantomNL** [comment]
Yesterday Protect 1.18.0 was released, has anyone already updated from 1.17.x and is able to confirm that the integration still works?

**Reply by briis** [reply]
I have been running the 1.18 beta for several weeks without any issues. Have not upgraded to final yet, but I have not found any breaking things during the beta versions.

**Comment by rhdny** [comment]
I'm on 1.18 final and everything works perfectly.
